projects/CompliantCSharpPackage/packages.lock.json

```diff
@@ -0,0 +1,25 @@
+{
```

Comment:
Does this file follow the .editorconfig preferences? (If not, we might want to exclude it from the BOM rule)

Reply:
At the very least my editor is not complaining about it when I do a clean rebuild

Reply:
They were created without BOM in any case